OmniBLE/PumpManagerUI/Views/OmniBLESettingsView.swift

```diff
@@ -256,8 +256,11 @@ struct OmniBLESettingsView: View  {
                     }
                     if let faultAction = viewModel.recoveryText {
                         Divider()
-                        Text(faultAction)
-                            .font(Font.footnote.weight(.semibold))
+                        VStack {
```

Comment:
I believe the enclosing VStack here can be removed. I'm guessing the .fixedSize modifier was necessary to fix some truncation issues, and we've had to use that in a number of places, so that's ok.

Reply:
Yes, I was originally using a VStack to align the text leading, but then I realized that it didn't behave exactly as intended when the screen was rotated and the need for alignment was better accommodated by a frame. I'd noticed that I left the VStack here after making the PR, but I will remove it.